pom.xml

```diff
@@ -27,16 +27,13 @@
     <openapi-generator.version>5.3.1</openapi-generator.version>
     <mapstruct.version>1.4.1.Final</mapstruct.version>
     <json.version>20200518</json.version>
-    <hazelcast.version>4.0.2</hazelcast.version>
     <spring-kafka.version>2.8.2</spring-kafka.version>
```

Comment:
This lines downgrades spring-kafka from 2.8.6 to 2.8.2.

Try `mvn dependency:tree -Dincludes=org.springframework.kafka:spring-kafka`:

```
[INFO] org.folio:mod-remote-storage:jar:1.6.0-SNAPSHOT
[INFO] \- org.springframework.kafka:spring-kafka:jar:2.8.2:compile
```

After removing `<spring-kafka.version>2.8.2</spring-kafka.version>`:

```
[INFO] org.folio:mod-remote-storage:jar:1.6.0-SNAPSHOT
[INFO] \- org.springframework.kafka:spring-kafka:jar:2.8.6:compile
```